modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -3,11 +3,9 @@ import {Meta, ArgsTable} from '@storybook/addon-docs';
 import {Grid, GridItem} from '@workday/canvas-kit-react/layout';
 // examples
 import {GridLayout} from './examples/Grid/GridLayout';
```

Comment:
Does the outer `<Grid gridAutoFlow="row"...` in the `GridLayout` example need to be there? Or can it replaced with `<Box padding="xs">`?

Reply:
It doesn't need to be a Grid component to wrap, it can be a Box component if that is something we should showcase.

Reply:
Cool, I replaced the outer `<Grid>` with `<Box padding="xs">` to simplify the example a bit.

Reply:
Awesome, thanks James!